x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/components/alert_summary_widget_ui.tsx

```diff
@@ -0,0 +1,87 @@
+ * 2.0.
+ */
+
+import { euiLightVars } from '@kbn/ui-theme';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { AlertsSummaryWidgetUIProps } from './types';
+
+export const AlertsSummaryWidgetUI = ({
+  active,
+  recovered,
+  timeRange,
+}: AlertsSummaryWidgetUIProps) => {
+  return (
+    <EuiPanel data-test-subj="ruleAlertsSummary" hasShadow={false} hasBorder>
+      <EuiFlexGroup direction="column">
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem grow={false}>
+              <EuiTitle size="xxs">
+                <h5>
+                  <FormattedMessage
+                    id="xpack.triggersActionsUI.sections.ruleDetails.alertsSummary.title"
+                    defaultMessage="Alerts"
+                  />
+                </h5>
+              </EuiTitle>
+              {!!timeRange && (
+                <>
+                  <EuiSpacer size="s" />
+                  <EuiText size="s" color="subdued">
+                    {timeRange}
+                  </EuiText>
+                </>
+              )}
+            </EuiFlexItem>
+
+            <EuiFlexItem>
+              <EuiFlexGroup gutterSize="s" alignItems="flexStart" responsive={false}>
+                <EuiFlexItem>
+                  <EuiText>
+                    <h3 data-test-subj="totalAlertsCount">{active + recovered}</h3>
+                  </EuiText>
+                  <EuiText size="xs" color="subdued">
+                    <FormattedMessage
+                      id="xpack.triggersActionsUI.sections.ruleDetails.alertsSummary.allAlertsLabel"
+                      defaultMessage="All"
+                    />
+                  </EuiText>
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiText color={euiLightVars.euiColorSuccessText}>
+                      <h3 data-test-subj="recoveredAlertsCount">{recovered}</h3>
+                    </EuiText>
+                  </EuiFlexItem>
+                  <EuiText size="xs" color="subdued">
+                    <FormattedMessage
+                      id="xpack.triggersActionsUI.sections.ruleDetails.rule.ruleSummary.recoveredLabel"
+                      defaultMessage="Recovered"
+                    />
+                  </EuiText>
+                </EuiFlexItem>
+                <EuiFlexItem>
+                  <EuiText color={euiLightVars.euiColorDangerText}>
+                    <h3 data-test-subj="activeAlertsCount">{active}</h3>
+                  </EuiText>
+                  <EuiText size="xs" color="subdued">
+                    <FormattedMessage
+                      id="xpack.triggersActionsUI.sections.ruleDetails.alertsSummary.activeLabel"
+                      defaultMessage="Currently active"
```

Comment:
❓ If I change the default value of a label, do I need to do something about the related translations or it will happen automatically? @simianhacker @XavierM

Reply:
if you only change the defaultMessage it will happen automatically. if you changed the ID, you will have to delete this ID in the translation. The process here, is that for every build candidate, they will re-create the translation.

Reply:
Got it, thanks!